### ISSUE TITLE
3.0.6 darklands not spawning

### ISSUE THREAD

**Issue by thejiji** [Darkosto/SevTech-Ages#1604]
version 3.0.6
mods removed : smooth fonts
java version 1.8.0 161

world generation set to default, creative mode enabled for flight to test

world from 3.0,4 broke when updating pack, so decided to start a new world to take advantage of worldgen fix's. after 6 hours of searching for a darlands for the chest quests, i decided to make a few test worlds to check.

generating a new world appears to not be spawning darklands biome, i have generated multiple worlds besides my initial world to test (5 worlds),  and flown search grids 30kx30k approx size without a single darklands biome spawning

expected behavior: normal spawning of darklands biome

**Comment by lordammon** [comment]
Have you checked either in the far North or South? They changed the worldgen for this version to have them in cold areas now.

**Comment by thejiji** [comment]
no, i will do that now, thanks for the heads up on the generation change!

**Comment by daddy77gaming** [comment]
#1569
